gnocchi/transformation.py

```diff
@@ -0,0 +1,60 @@
+# -*- encoding: utf-8 -*-
+#
+# Copyright © 2017 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import collections
+import functools
+
+import pyparsing as pp
+
+from gnocchi import utils
+
+
+Transformation = collections.namedtuple('Transformation', ["method", "args"])
```

Comment:
define this in `gnocchi.carbonara` because that's where it's really used in the end?

Reply:
done